style.css

```diff
@@ -122,16 +122,43 @@
 /*********************************** CYCLE SECTION *************************************/
 /***************************************************************************************/
 
-.custom-cycle-body {
+.cycle-service {
     padding: 5rem 1rem;
+    display: flex;
     background-image: linear-gradient(#a5a2a2, #e4e0e0);
 }
+.cycle-col {
+    margin: 0 2rem;
+    width: 45%;
+}
+
+.cycle-col__image {
+    margin-top: 3rem;
+}
 
-.cycle-heading {
+.cycle-col__title {
+    margin: 1.5rem;
     font-size: 3rem;
-    padding: 4rem;
+    font-weight: bold;
 }
 
-.cycle-text {
-    padding: 4rem;
+.cycle-col__text {
+    margin: 1rem;
+    padding: 0 1rem;
+    font-size: 1.2rem;
+    text-align: justify;
+}
+
+.cycle-col__link_button {
+    margin: 4rem auto 2rem;
```

Comment:
avoid using margin-top

Reply:
okay, we avoided it in this part of styles